### PR TITLE
Documentation/*: change github url, import path from coreos to etcd-io

### PR DESCRIPTION
**Pull request by vimalk78**
Please read https://github.com/etcd-io/etcd/blob/master/CONTRIBUTING.md#contribution-flow.

### CODE REVIEW

File: Documentation/dev-guide/grpc_naming.md [hunk 1]
```diff
@@ -8,8 +8,8 @@ The etcd client provides a gRPC resolver for resolving gRPC endpoints with an et
 
 ```go
 import (
-	"github.com/coreos/etcd/clientv3"
-	etcdnaming "github.com/coreos/etcd/clientv3/naming"
+	"github.com/etcd-io/etcd/clientv3"
+	etcdnaming "github.com/etcd-io/etcd/clientv3/naming"
 
 	"google.golang.org/grpc"
 )
```
Comment (hexfusion) [inline Documentation/dev-guide/grpc_naming.md]: Hi @vimalk78 import will now be `go.etcd.io` like https://github.com/etcd-io/etcd/commit/fced933294e86899842be026058a94b9baeefc97

Comment (vimalk78) [reply Documentation/dev-guide/grpc_naming.md]: thanks @hexfusion , missed it. done !

**Comment by hexfusion** [comment]
lgtm, thanks @vimalk78!

**Comment by vimalk78** [comment]
i guess no need to update `Documentation/upgrades/upgrade_*.md` for the new import path ?

**Comment by hexfusion** [comment]
@vimalk78 good question to this point all point releases for 3.{1,2,3} use coreos org. I believe the next release cycle will bring in new vanity import? But for sure 3.4 will. So upgrade_3_{4,5}.md should get changed and maybe @gyuho can give input on others.

**Comment by vimalk78** [comment]
CI check failed for this test case.
any suggestion?
```
=== RUN   TestLeasingReconnectTxn
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x60 pc=0xaf84fd]
goroutine 24847 [running]:
go.etcd.io/etcd/vendor/google.golang.org/grpc.(*csAttempt).sendMsg(0xc000a38840, 0x14a6940, 0xc00035ad10, 0xc0054ceb5b, 0x5, 0x5, 0xc0054ceb50, 0xb, 0xb, 0xc0054ceb50, ...)
	/go/src/go.etcd.io/etcd/vendor/google.golang.org/grpc/stream.go:713 +0x14d
go.etcd.io/etcd/vendor/google.golang.org/grpc.(*clientStream).SendMsg.func2(0xc000a38840, 0x48, 0x48)
	/go/src/go.etcd.io/etcd/vendor/google.golang.org/grpc/stream.go:637 +0x186
go.etcd.io/etcd/vendor/google.golang.org/grpc.(*clientStream).withRetry(0xc0003ecb40, 0xc005aab9f0, 0xc0000bfaa0, 0xc0054ceb5b, 0x5)
	/go/src/go.etcd.io/etcd/vendor/google.golang.org/grpc/stream.go:530 +0x431
go.etcd.io/etcd/vendor/google.golang.org/grpc.(*clientStream).SendMsg(0xc0003ecb40, 0x14a6940, 0xc00035ad10, 0x0, 0x0)
	/go/src/go.etcd.io/etcd/vendor/google.golang.org/grpc/stream.go:643 +0x80c
go.etcd.io/etcd/etcdserver/etcdserverpb.(*watchWatchClient).Send(0xc00035ad00, 0xc00035ad10, 0x0, 0x2057738)
	/go/src/go.etcd.io/etcd/etcdserver/etcdserverpb/rpc.pb.go:3618 +0x6b
go.etcd.io/etcd/clientv3.(*watchGrpcStream).run(0xc0013369c0)
	/go/src/go.etcd.io/etcd/clientv3/watch.go:542 +0xd84
created by go.etcd.io/etcd/clientv3.(*watcher).newWatcherGrpcStream
	/go/src/go.etcd.io/etcd/clientv3/watch.go:277 +0x58c
FAIL	go.etcd.io/etcd/clientv3/integration	133.109s
```

**Pull request merged**